setup.py

```diff
@@ -2,17 +2,23 @@
 # -*- coding: utf-8 -*-
 # Licensed under a 3-clause BSD style license, see LICENSE.
 
+import os.path
+
 from setuptools import setup, find_packages
 
 with open('README.md') as readme_file:
     readme = readme_file.read()
 
+def get_version():
+    g = {}
+    exec(open(os.path.join("skhep_testdata", "version.py")).read(), g)
+    return g["__version__"]
```

Comment:
Is there a reason why we couldn't do:
```python
import skhep.version as v
return v.__version__
```

Reply:
I'm not sure since this is part of the setup.py file, hence I would not assume too much on imports - I did not find myself the working solution above. I would just leave it as is, then.